tests/test_env.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: MIT
+
+import os
+import subprocess
+import sys
+import sysconfig
+
+import build.env
+
+
+def test_isolated_environment(mocker):
+    with build.env.IsolatedEnvironment() as env:
+        assert os.environ['PATH'] == os.path.join(env._path, 'bin')
```

Comment:
bin, what about windows where it's scripts?

Reply:
We get the scripts path from `sysconfig`, so it should be right. We are only testing on Linux right now, so I missed this.

Reply:
We really should test on all platforms: Linux, Windows, macOs. There's no working around this if we want this to move under PyPa, so I consider this major blocking.

Reply:
I agree, I will do it in a future PR.